### PR TITLE
Feature/mat 6077 qdm attr in table

### PR DESCRIPTION
**Pull request by RohitKandimalla**
## MADiE PR

Jira Ticket: [MAT-6077](https://jira.cms.gov/browse/MAT-6077)
(Optional) Related Tickets:

### Summary

image

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included
* [x] No extemporaneous files are included (i.e Complied files or testing results)
* [x] This PR is into the **correct branch**.
* [ ] All Documentation as needed for this PR is Complete (or noted in a TODO or other Ticket)
* [ ] Any breaking changes or failing automation are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package)
* [ ] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo)

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
*  The tests appropriately test the new code, including edge cases
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads

### CODE REVIEW

File: src/components/editTestCase/qdm/LeftPanel/ElementsTab/Elements/DataElementsTable/DataElementsTable.tsx [hunk 1]
```diff
@@ -11,20 +12,46 @@ import {
 import useQdmExecutionContext from "../../../../../../routes/qdm/useQdmExecutionContext";
 import TimingCell from "./TimingCell";
 import DatElementActions from "./DatElementActions";
+import ExpandMoreIcon from "@mui/icons-material/ExpandMore";
+import {
+  generateAttributesToDisplay,
+  SKIP_ATTRIBUTES,
+  stringifyValue,
```
Comment (adongare) [inline src/components/editTestCase/qdm/LeftPanel/ElementsTab/Elements/DataElementsTable/DataElementsTable.tsx]: looks like unused imports

Comment (RohitKandimalla) [reply src/components/editTestCase/qdm/LeftPanel/ElementsTab/Elements/DataElementsTable/DataElementsTable.tsx]: Removed them

File: src/util/QdmAttributeHelpers.ts [hunk 2]
```diff
@@ -86,6 +87,67 @@ export const determineAttributeTypeList = (path, info) => {
...
 };
 
+// This is specific to DataElements Table as multiple data types from same attribute has to be displayed in same cell
+export const generateAttributesToDisplay = (
+  dataElement,
+  dataElements,
+  codeSystemMap: any
+) => {
+  const dataElementClass = getDataElementClass(dataElement);
+  const modeledEl = new dataElementClass(dataElement);
```
Comment (adongare) [inline src/util/QdmAttributeHelpers.ts:97]: this is just creating a new instance of data element. is there anything that is not available in `dataElement` that forces you to create a new instance?

Comment (RohitKandimalla) [reply src/util/QdmAttributeHelpers.ts:97]: Yes, we need Schema, thats not available in our JSON

**Review by adongare — APPROVED** [review]

**Pull request merged**